# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtk-perl10/gendefs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./tools/gendefs.pl~	Tue Mar  9 19:46:14 1999
+++ ./tools/gendefs.pl	Tue Apr 27 16:08:13 1999
@@ -517,6 +517,10 @@
 #undef _
 #endif
 
+#ifdef dirty
+#undef dirty
+#endif
+
 #ifndef Perl${FilePrefix}DeclareFunc
 #include "Perl${FilePrefix}Int.h"
 #endif
